footer.md

```diff
@@ -0,0 +1,54 @@
+% Here we're using `grid` to imply grid semantics -- the actual
+% formatting (including columns) is handled by CSS
```

Comment:
I recommend adding a URL that has a link back to the docs for people that land here and want to learn more (maybe the section navigation page?)